src/utils/transaction.js

```diff
@@ -93,9 +93,9 @@ const transformTransaction = ({
     case unlockToken: {
       transformedTransaction.asset = {
         unlockObjects: asset.unlockObjects.map(unlockObject => ({
-          delegateAddress: getBase32AddressFromAddress(unlockObject.delegateAddress),
-          amount: Number(unlockObject.amount),
-          unvoteHeight: unlockObject.height.start,
+          delegateAddress: getBase32AddressFromAddress(Buffer.from(unlockObject.delegateAddress, 'hex')),
+          amount: convertBigIntToString(unlockObject.amount),
+          unvoteHeight: unlockObject.unvoteHeight,
```

Comment:
Isn't this supposed to be `unlockobject.height.start`? This is what I remember but may be its changed now. Can you please confirm again?

Reply:
`e423209a2ae3251970e748b040c2a6f67c9f8f04d8997b52493c770368a4ce50` tx was done using a regular account and `699fb0682f9389287d0c081f39cf8034435290332f4c040671bce3bd245c9a9a` was done using a multisig account. Both with the current code base. We were converting `unlockObject.height.start` to `unvoteHeight` in `getUnlockableUnlockObjects` anyways.